rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -866,9 +875,12 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
 		const bool is_vulkan = (text == r_creator->Vulkan.name);
 		ui->asyncTextureStreaming->setEnabled(is_vulkan);
 		ui->vulkansched->setEnabled(is_vulkan);
-		ui->fsrUpscalingEnable->setEnabled(is_vulkan);
-		ui->fsrSharpeningStrength->setEnabled(is_vulkan);
-		ui->fsrSharpeningStrengthReset->setEnabled(is_vulkan);
+
+		const QVariantList var_list = ui->outputScalingMode->itemData(ui->outputScalingMode->currentIndex()).toList();
+		ensure(var_list.size() == 2 && var_list[1].canConvert<int>());
+		const bool fsr_selected = static_cast<output_scaling_mode>(var_list[1].toInt()) == output_scaling_mode::fsr;
+		ui->fsrSharpeningStrength->setEnabled(is_vulkan && fsr_selected);
+		ui->fsrSharpeningStrengthReset->setEnabled(is_vulkan && fsr_selected);
```

Comment:
then remove this and add an additional connect:
`connect(ui->outputScalingMode, QOverload::of(&QComboBox::currentIndexChanged), this, enableFsrSettings);`
and call
`enableFsrSettings();`
once in the end to ensure they are enabled/disabled